src/parser.ts

```diff
@@ -1,25 +1,89 @@
+import * as vscode from "vscode";
+
 function normalizeQuery(query: string) {
   return query.replace(/''/g, "'");
 }
 
+function doesFloatParamExistInDeclaration(parameterDeclaration:string, paramName:string){  
+  let paramWithUpperType = paramName + " FLOAT";
```

Comment:
In order to support all different cases like `FLoat`, I can suggest converting to whole parameter declaration string to lowercase and then we can only check the against the `float`

Reply:
Done